ui/component/liquidateSupports/view.jsx

```diff
@@ -0,0 +1,133 @@
+  supportsBalance: number,
+  tipsBalance: number,
+  claim: any,
+  metaData: any,
+  handleClose: () => void,
+  abandonSupportForClaim: (string, boolean | string, boolean) => any,
+};
+
+const LiquidateSupports = (props: Props) => {
+  const { claim, abandonSupportForClaim, handleClose } = props;
+  const [previewBalance, setPreviewBalance] = useState(1);
+  const [amount, setAmount] = useState(0);
+  const [error, setError] = useState(false);
+  const [message, setMessage] = useState('Amount to unlock');
+  const keep =
+    Number(amount) < previewBalance ? Number.parseFloat(String(previewBalance - Number(amount))).toFixed(8) : false;
+  const claimId = claim && claim.claim_id;
+
+  const MESSAGE_NUMBER = __('Amount must be a number');
+  const MESSAGE_BLANK = __('Amount cannot be blank');
+  const MESSAGE_STAKED = __('Your content will do better with more staked on it');
+  const MESSAGE_MAX = __('Amount cannot be more than available');
+  const MESSAGE_ZERO = __('Amount cannot be zero');
+  const MESSAGE_DEFAULT = __('Amount to unlock');
+  const MESSAGE_CLOSE = __(`She's about to close up the library!`);
+  useEffect(() => {
+    if (claimId && abandonSupportForClaim) {
+      abandonSupportForClaim(claimId, false, true).then(r => {
+        setPreviewBalance(r.total_input);
+      });
+    }
+  }, [abandonSupportForClaim, claimId, setPreviewBalance]);
+
+  function handleSubmit() {
+    abandonSupportForClaim(claimId, keep, false);
+    handleClose();
+  }
+
+  function handleChange(a) {
+    if (a === undefined || isNaN(Number(a))) {
+      setMessage(MESSAGE_NUMBER);
+      setError(true);
+      setAmount('');
+    } else if (a === '') {
+      setAmount('');
+      setError(true);
+      setMessage(MESSAGE_BLANK);
+    } else if (Number(a) > previewBalance) {
+      setMessage(MESSAGE_MAX);
+      setError(false);
+    } else if (Number(a) === Number(previewBalance)) {
+      setMessage(MESSAGE_CLOSE);
+      setAmount(a);
+      setError(false);
+    } else if (Number(a) > previewBalance / 2) {
+      setMessage(MESSAGE_STAKED);
+      setAmount(a);
+      setError(false);
+    } else if (a === '0') {
+      setMessage(MESSAGE_ZERO);
+      setAmount(a);
+      setError(true);
+    } else {
+      setMessage(MESSAGE_DEFAULT);
+      setAmount(a);
+      setError(false);
+    }
+  }
+
+  return (
+    <Card
+      title={__('Unlock Tips')}
+      subtitle={
+        <React.Fragment>
+          <div className="section__flex">
+            <Icon sectionIcon icon={ICONS.LOCK} />
+            <div>
+              <h2 className="section__title--small">
+                <strong>
+                  <CreditAmount badge={false} amount={previewBalance || '...'} precision={2} />
+                </strong>{' '}
+                {__('available to unlock.')}
+              </h2>
+            </div>
+          </div>
+          <p>
+            You can unlock all or some of total supports. This will free up LBC, but reduce content standings. Fees
```

Comment:
i18n

Reply:
You can unlock all or some of the locked LBC. The more you unlock, the less trusted your content is. Learn more.

Reply:
Learn More -> /faq/tipping